src/librustc/ty/layout.rs

```diff
@@ -312,39 +314,36 @@ impl Align {
             }
             if bytes != 1 {
                 Err(format!("`{}` is not a power of 2", align))
-            } else if pow > 0x0f {
-                Err(format!("`{}` is too large", align))
```

Comment:
The parsing is for the user-facing value. Here we must protect any internal guarantees.